WalletWasabi.Backend/Controllers/ChaumianCoinJoinController.cs

```diff
@@ -222,19 +223,12 @@ public async Task<IActionResult> PostInputsAsync([FromBody, Required]InputsReque
 						// Check if unconfirmed.
 						if (getTxOutResponse.Confirmations <= 0)
 						{
+							allInputsConfirmed = false;
```

Comment:
Maybe better to move this after the if statement?

Reply:
ack